R/wflow_toc.R

```diff
@@ -19,9 +19,6 @@
 #' \item Check that the system keyboard is writable. Run
 #' \code{\link[clipr]{clipr_available}} and \code{\link[clipr:clipr_available]{dr_clipr}}.
 #'
-#' \item If it's still not working, set \code{keyboard = FALSE} to send the
-#' table of contents to the R console to manually copy-paste.
```

Comment:
This was a typo. It should have said `\code{clipboard = FALSE}`. Could you please add back this line and then replace `keyboard` with `clipboard`?